### DISCUSSION TITLE
Hope to achieve Lua5.3.4!

### DISCUSSION THREAD

**Discussion by jefurry** [yuin/gopher-lua#373] [Ideas]
Hope to achieve Lua5.3.4! thanks.

**Comment by ghost** [comment]
This is unlikely to happen, as gopher-lua is specifically a Lua 5.1 implementation.

I did, however, find a 5.3 implementation here: https://github.com/milochristiansen/lua

**Comment by yuin** [comment]
@jefurry @bontibon @technosophos Now I working on new version of GopherLua that supports Lua5.3 specs.

You can see the working codes in [/5.3 directory at 5.3 branch](https://github.com/yuin/gopher-lua/tree/5.3/5.3).

**Comment by Chillance** [comment]
@yuin This is sweet. Any idea when you think this will be ready and released?

**Comment by Hades32** [comment]
@yuin this is nice! I'd currently mostly need a few math functions that are missing in older Lua versions. Would you accept PRs just containing these?

**Comment by yuin** [comment]
@Hades32 Sorry, I haven't gotten around to this yet.
I would be happy to see your PR. But I concerns about number specification difference between 5.1 and 5.3.
Lua5.1 has only one number type: Number. Lua5.3 has two number types: Float and Integer.
This may affects math functions. For example, sum operation between Float and Integer results Float, but Integer and Integer results Integer.
So I will have to add new type for GopherLua and to modify basic arithmetic operations. This maybe conflict with your PR.

**Comment by jefurry** [comment]
@yuin It is great! When is it released?
